packages/plugin-calendar/src/config.js

```diff
@@ -0,0 +1,21 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+export default {
+  calendar: {
+    /**
+     * Set to true to enable the terrible implementation
+     * @type {Boolean}
+     */
+    enableTerribleImplementation: false
+  },
+  device: {
+    preDiscoveryServices: {
+      calendarServiceUrl: process.env.CALENDAR_API_SERVICE_URL || `https://calendar-a.wbx2.com/calendar/api/v1`,
+      whistlerServiceUrl: process.env.WHISTLER_API_SERVICE_URL || `http://internal-testing-services.wbx2.com:8084/api/v1`
```

Comment:
whistler is not a prediscovery service. also, internal urls should not be in the sdk.